ras/sosreport.py

```diff
@@ -181,17 +180,15 @@ def test_others(self):
         self.run_cmd("sosreport --batch --tmp-dir=%s --debug" % directory_name, None)
 
         self.run_cmd("sosreport --batch --tmp-dir=%s --config-file=/etc/sos.conf" % directory_name)
-        file_name = self.run_cmd_out("sosreport --batch --tmp-dir=%s --tmp-dir=/root | "
-                                     "grep root | tail -1" % directory_name).strip()
+        file_name = self.run_cmd_out("sosreport --batch --tmp-dir=%s | "
+                                     "grep %s | tail -1" % (directory_name, directory_name)).strip()
         if not os.path.exists(file_name):
             self.is_fail += 1
             self.log.info("--tmp-dir option failed")
 
         dir_name = self.run_cmd_out("sosreport --no-report --batch --build | "
-                                    "grep located | "
-                                    "cut -d':' -f2").strip()
-        sosreport_dir = os.path.join(dir_name, 'sos_reports')
-        if os.listdir(sosreport_dir) != []:
+                                    "grep '/var/tmp/sosreport'").strip()
```

Comment:
will it always be `/var/tmp` or it could be `/tmp` sometimes?

Reply:
it will be always /var/tmp unless we specify different folder explicitly.